vela/client.go

```diff
@@ -388,11 +394,13 @@ func (r *Response) populatePageValues() {
 // u is the URL you want to call.
 // body is the HTTP body.
 // v is the HTTP response.
+// headers is a map of the HTTP headers.
 //
 // For more information read https://github.com/google/go-github/issues/234
-func (c *Client) Call(method, u string, body interface{}, v interface{}) (*Response, error) {
+// nolint: lll // ignore long line length due to variable names
+func (c *Client) Call(method, u string, body interface{}, v interface{}, headers map[string]string) (*Response, error) {
```

Comment:
Do you think it's worth exploring adding a separate function here instead of this?

Thinking it could be named something like `CallWithHeaders`?

The idea would be to leave the generic `Call` function as is.

This function gets used in so many different files/functions which is why I bring it up.

I'm not sure there is value in modifying that function for a single use-case like this.

AFAICT, every other place where `Call` is used we are passing `nil` for the `headers`.

This indicates to me that perhaps we should have a separate function?

Reply:
should be addressed in latest changes